low-level/actuators/impl/centos_7/systemd_service.py

```diff
@@ -165,3 +179,21 @@ def _get_active_substate(self):
         return self._proxy.Get('org.freedesktop.systemd1.Unit',
                                 'SubState',
                                 dbus_interface='org.freedesktop.DBus.Properties')
+
+    def is_service_enabled(self, service_name):

```

Comment:
Currently cortx-utils Service class is taking care of all service related actions (enable, start, stop, restart, etc,.)
But it doesn't have code for for service status. If this can be put there, it would be more useful.

Reply:
For start/stop/restart used cortx utils code, for enable/disable cortx.util code was giving "Interactive Authorization required" exceeption for sspl-ll user, hence for enable/disable using the code which is already present in systemd actautor class.